test/e2e/testsuite/testsuite.go

```diff
@@ -961,11 +977,6 @@ func PodmanCommandIsAvailable() error {
 		csshk = filepath.Join(userHomeDir, ".crc/machines/crc/id_ed25519")
 		dh = "npipe:////./pipe/crc-podman"
 	}
-	/* change linux default to user
-	if runtime.GOOS == "linux" {
-		ch = "ssh://core@192.168.130.11:22/run/user/1000/podman/podman.sock"
-	}
-	*/
```

Comment:
May be commenting out of this shouldn't be part of first commit and this commit remove it completely.